components/Footer/Footer.js

```diff
@@ -3,12 +3,24 @@ import styled from 'styled-components'
 import FooterContentArea from './FooterContentArea'
 import Map from './Map'
 import Link from 'next/link'
+import GoogleLogoPNG from '../../assets/png/footer/google-logo.png'
 import { useMediaQuery } from 'react-responsive'
 
 export default function Footer() {
   const isDesktop = useMediaQuery({ query: '(max-width: 841px)' })
   return (
     <Wrapper>
+      <GoogleRezession>
```

Comment:
Just a little typo. 

```suggestion
      <GoogleRezension>
```

Reply:
It's not a typo if the style component is named as this ;-) But yes, will change this. Thx

Reply:
Oh, I didn't see the style component 😅